vscode_extension/src/LanguageClient.ts

```diff
@@ -70,6 +70,24 @@ export function shimLanguageClient(
   };
 }
 
+/**
+ * Handles short-circuiting format-on-save requests when either the client or server is
+ * configured to turn them off.
```

Comment:
```suggestion
 * configured to turn them off.
 * 
 * We shim this here in the extension so that it doesn't matter which version of
 * Sorbet is installed. (An alternative would be to shim it in Sorbet, but this would
 * break if a newer VS Code extension (passing formatOnSave: true) is used to drive an
 * older Sorbet that doesn't respond to textDocument/formatting with no-op responses.)
```